site/sfguides/src/getting_started_with_cortex_search_and_microsoft_fabric/getting_started_with_cortex_search_and_microsoft_fabric.md

```diff
@@ -0,0 +1,309 @@
+        limit (int): The maximum number of results to return.
+
+    Returns:
+        dict: The search results in JSON format.
+    """
+    try:
+        # create a root as the entry point for all objects
+        root = Root(connection)
+        
+        # perform the search
+        response = (
+            root.databases[CONNECTION_PARAMETERS["database"]]
+            .schemas[CONNECTION_PARAMETERS["schema"]]
+            .cortex_search_services[svc]
+            .search(
+                query,
+                columns,
+                filter_type,
+                limit=limit
+            )
+        )
+
+        print(f"Received response with `request_id`: {response.request_id}")
+        results = json.dumps(response.results, indent=5)
+        print(results)
+        return results
+    except Exception as e:
+        print(f"An error occurred: {e}")
+```
+
+Let's now run the function and place the results into a pandas dataframe.
+
+```python
+import pandas as pd
+results = search_records(connection, query, columns, svc, filter_type, limit)
+l = json.loads(results)
+df = pd.DataFrame(l)
+```
+We have created a function that calls the service filtering only on Emergency admissions and using plain text to query the service and return records in which the discharge notes indicate an allergy to Penicillin. Then created a data frame with those records.
+
+Now, let's take some time to review the results with some of Fabric Notebooks interactive EDA capabilities. Run the below code and click through the data to view the results. If you click through records on the "TEXT" field you can check to see if we're actually returning records where patients are allergic to Penicillin.
+
+```python
+display(df)
+```
+
+![](assets/displaycheck.png)
+
+Click through additional fields to view the values in the table.
+
+Now let's experiment with some more EDA in the results. Execute the below code in your Fabric Notebook.
+
+```python
+display(df, summary=True)
+```
+
+You can click through the different fields to see how patients allergic to Penicillin are separated across different categories.
+
+![](assets/edacheck.png)
+
+To learn more about Fabric Notebook functionality check out the documentation [here](https://learn.microsoft.com/en-us/fabric/data-engineering/author-execute-notebook).
+
+Finally, we can write the results of the Search query back to a table in Snowflake.
+
+```python
+from snowflake.connector.pandas_tools import write_pandas
+
+# Write the DataFrame to Snowflake
+success, nchunks, nrows, _ = write_pandas(connection, df, 'ALLERGY_TABLE')
+```
+
+You can head back to Snowflake to verify the table was created. And you can do with this whatever you would like...analyze the results in Snowflake, connect Power BI to it or build models with Cortex, SnowflakeML, Azure Openai and AzureML!
+
+![](assets/allergytable.png)
+
+
+<!-- ------------------------ -->
+## Conclusion and Additional Considerations
```

Comment:
@sfc-gh-mmarzillo Can you please rename this to "Conclusion And Resources"?

Reply:
done